src/OpenTelemetry/CoreSdkForwarding/README.md

```diff
@@ -0,0 +1,35 @@
+# OpenTelemetry - .NET Metrics
+
+This sample shows how to configure the SDK to forward metrics from the Core SDK.
+
+The main advantage over using .NET metrics is simplicity; additionally, there is no need to take a dependency on the .NET OpenTelemetry libraries.
+
+Note that in order to set up tracing, .NET OpenTelemetry must be used. See the [DotNetMetrics](../DotNetMetrics) sample for an example of how to set up tracing.
```

Comment:
This is a bit confusing. This sample should probably add OTel tracing interceptor too. Many of our users setup OTel interceptor tracing w/ meter and such and still using the Core SDK-based OTel metrics instead of the .NET custom meter one.

Reply:
Fixed